app/controllers/ask-for-more-time/ask-for-more-time.ts

```diff
@@ -160,7 +160,7 @@ function postCheckAndSend(updateAppealService: UpdateAppealService) {
   return async (req: Request, res: Response, next: NextFunction) => {
     try {
       req.session.appeal.askForMoreTime.status = 'submitted';
-      req.session.appeal.askForMoreTime.requestedDate = nowIsoDate();
+      req.session.appeal.askForMoreTime.requestDate = nowIsoDate();
```

Comment:
I noticed Lewis had changed this too but the field in CCD is requestedDate so why have a different name? This will still work though

Reply:
So the original intention of the `requestDate` was to log when the time extension was requested but this seem to have changed and was being used as the new date set by TCW. So in order to differentiate between the two, we changed it to: 
`requestDate` When the request was made.
`decisionOutcomeDate` The date set by TCW if granted if not it defaults to the old direction due date.